test/Microsoft.AspNetCore.Razor.Language.Test/DefaultRazorTagHelperBinderPhaseTest.cs

```diff
@@ -883,11 +908,11 @@ public static TheoryData ProcessTagHelperPrefixData
         {
             // Arrange
             var errorSink = new ErrorSink();
-            var pass = new TagHelperBinderSyntaxTreePass();
+            var phase = new DefaultRazorTagHelperBinderPhase();
             var document = RazorCodeDocument.Create(new DefaultRazorSourceDocument("Test content", encoding: Encoding.UTF8, fileName: "TestFile"));
 
             // Act
-            var prefix = pass.ProcessTagHelperPrefix(((IEnumerable<TagHelperDirectiveDescriptor>)directiveDescriptors).ToList(), document, errorSink);
+            var prefix = phase.ProcessTagHelperPrefix(((IEnumerable<TagHelperDirectiveDescriptor>)directiveDescriptors).ToList(), document, errorSink);
```

Comment:
You can put `IEnumerable<TagHelperDirectiveDescriptor>` in the parameter list and get rid of this cast. This was an old workaround for when THD was internal.